### PR TITLE
Tp/allowusersession

### PR DESCRIPTION
**Pull request by tom-parker**
cc @commuterjoy @georgecrawford

### CODE REVIEW

**Comment by georgecrawford** [comment]
Looks good. In theory, I'd prefer to use `system.source` for the whitelist, as it's supposed to be a 'machine-readable' identifier for the app which sent the request. However, we never defined a format for this, and most products are now using `o-tracking`, so I guess `context.product` is a better filter for now.

**Comment by tom-parker** [comment]
Ok. I think the whitelist will only be a temporary thing anyway whilst we're making Ammit more resilient and FT.com doesn't need AB segmentation.

File: server/transforms/ab-api.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ require('isomorphic-fetch');
 const metrics = require('next-metrics');
 const logger = require('ft-next-logger').logger;
 const reportError = require('../utils/report-error');
+const appWhiteList = ['next', 'FT app'];
```
Comment (commuterjoy) [inline server/transforms/ab-api.js]: perhaps call this `productWhitelist` as that's where you pluck the value from.

Comment (georgecrawford) [inline server/transforms/ab-api.js]: yep, good

**Comment by commuterjoy** [comment]
+1 looks good

**Pull request merged**